sumpy/symbolic.py

```diff
@@ -268,12 +279,14 @@ def vector_xreplace(expr, from_vec, to_vec):
     return expr.xreplace(substs)
 
 
-def find_power_of(base, prod):
+def find_power_of(base: sym.Basic, prod: sym.Basic) -> int | sym.Basic:
     remdr = sym.Wild("remdr")
     power = sym.Wild("power")
+
     result = prod.match(remdr*base**power)
     if result is None:
         return 0
+
     return result[power]
```

Comment:
This doesn't seem used and it also doesn't work with `symengine` as far as I can tell?